jobs/rocoto/atmos_products.sh

```diff
@@ -22,7 +22,11 @@ IFS='_' read -ra fhrs <<< "${FHRLST//f}" # strip off the 'f's and convert to arr
 #---------------------------------------------------------------
 # Execute the JJOB
 for fhr in "${fhrs[@]}"; do
-    export FORECAST_HOUR=$(( 10#${fhr} ))
+    # The analysis fhr is -001.  Performing math on negative, leading 0 integers is tricky.
+    # The negative needs to be in front of "10#", so do some regex magic to make it happen.
+    fhr="10#${fhr}"
+    fhr=$( echo "${fhr}" | sed "s/10#-/-10#/" )
```

Comment:
This will get rid of the shellcheck error
```suggestion
    fhr=${fhr//10#-/-10#}
```